src/main/resources/db/changelog/changes/v1.0.0/drv_item_callnumber_location_create.xml

```diff
@@ -0,0 +1,78 @@
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+  <changeSet id="create-view-item-callnumber-location" author="mweaver@ebsco.com">
+    <preConditions onFail="CONTINUE">
+      <viewExists viewName="src_inventory_item"/>
+      <viewExists viewName="src_inventory_material_type"/>
+      <viewExists viewName="src_inventory_location"/>
+      <viewExists viewName="src_inventory_loclibrary"/>
+      <viewExists viewName="src_inventory_call_number_type"/>
+      <sqlCheck expectedResult="1">
+        SELECT COUNT(*) FROM pg_matviews WHERE schemaname = '${tenant_id}_mod_fqm_manager'AND matviewname = 'drv_inventory_item_status';
+      </sqlCheck>
+    </preConditions>
+    <createView
+      replaceIfExists="true"
+      viewName="drv_item_callnumber_location">
+      SELECT src_inventory_item.id,
+        src_inventory_item.holdingsrecordid                                                           AS item_holdings_record_id,
+        src_inventory_item.jsonb ->> 'hrid'::text                                                     AS item_hrid,
+        src_inventory_item.jsonb ->> 'itemLevelCallNumber'::text                                      AS item_level_call_number,
+        src_inventory_item.jsonb ->> 'itemLevelCallNumberTypeId'::text                                AS item_level_call_number_typeid,
+        call_item_number_type_ref_data.item_level_call_number_type_name,
+        (src_inventory_item.jsonb -> 'effectiveCallNumberComponents'::text) ->>'callNumber'::text     AS item_effective_call_number,
+        (src_inventory_item.jsonb -> 'effectiveCallNumberComponents'::text) ->>'typeId'::text         AS item_effective_call_number_typeId,
+        call_number_type_ref_data.item_effective_call_number_type_name,
+        loclib_ref_data.id                                                                            AS item_effective_library_id,
+        loclib_ref_data.item_effective_library_name,
+        loclib_ref_data.item_effective_library_code,
+        "left"(lower(${tenant_id}_mod_inventory_storage.f_unaccent((src_inventory_item.jsonb -> 'status'::text) ->> 'name'::text)), 600) AS item_status,
```

Comment:
This is the only real change in this PR. I reformatted the view when moving it to XML, but didn't really change anything other than adding the `${tenant_id}_mod_inventory_storage.` to the `f_unaccent()` call here.

Reply:
Technically, the preconditions are different, too. I stripped those down to just the views actually used by drv_item_callnumber_location